cloudbuild.yaml

```diff
@@ -0,0 +1,25 @@
+timeout: 1200s
+options:
+  volumes:
+  - name: go-modules
+    path: /go
+  env:
+  - GOPROXY=https://proxy.golang.org
+  - PROJECT_ROOT=github.com/transparency-dev/witness
+  - GOPATH=/go
+
+# Cloud Build logs sent to GCS bucket
+logsBucket: 'gs://trillian-cloudbuild-logs'
+
+steps:
+- id: 'lint'
+  name: "golangci/golangci-lint"
+  args: ["golangci-lint","run"]
+
+- id: 'unit tests'
+  name: 'golang:1.20'
```

Comment:
Any reason for 1.20 here? The go.mod is 1.19 so I'd be tempted to sync them?

Reply:
Sure, 0.01 less golang done!